src/main/kotlin/race/Car.kt

```diff
@@ -1,22 +1,16 @@
 package race
 
-class Car(val name: String, var movements: MutableList<Boolean> = mutableListOf()) {
+class Car(val name: String) {
+    val movements: MutableList<Boolean> = mutableListOf()
```

Comment:
여전히 외부에서 movements의 값을 조작할 수 있는 형태입니다.
```kotlin
val car = Car("name")
car.movements.clear() // 외부에서 수정이 가능해 캡슐화가 깨지고 있습니다.
```

초기 움직인 값을 세팅할 수도 있고, 외부에서는 movements를 수정할 수 없게 만든다면, 아래 처럼 만들어볼 수 있어요 :)
또 다른 방법들이 있을 수 있으니 취향에 맞게 작성해보면 좋겠어요 😀

```suggestion
class Car(val name: String, initialMovements: List<Boolean>) {
    private val movements: MutableList<Boolean> = mutableListOf()
```

또, 이번 미션의 주제는 아니지만, movements는 List이니 강의에서 배워오셨던 일급 컬렉션을 활용해볼 수도 있겠네요 🙂

Reply:
조언 감사합니다!